City-of-Helsinki/berth-reservations-admin
# VEN-625 | Create customer message modal

## Description :sparkles:
- Create message modal form and initial layout

### Closes :no_good_woman:
https://helsinkisolutionoffice.atlassian.net/browse/VEN-625
## Testing :alembic:

1. Go to customers
2. Select customer(s)
3. Hit "Lähetä viesti" button
4. Message modal should be shown and validated on submit, fields are mandatory
5. Submit should popup alert which shows the data

Out of scope: we should probably disable the "Lähetä viesti" if no customers selected

### Automated tests :gear:️
Component render test `CustomerListTableTools.test.tsx`
